### DISCUSSION TITLE
Offload 3rd party VSTi's to WAVES Soundgrid Server One-C

### DISCUSSION THREAD

**Discussion by melodiacman** [apohl79/audiogridder#1041] [General]
Will it be possible to use AG to offload 3rd party VSTi's to the Waves Soundgrid Server ONE-C?

**Comment by apohl79** [comment]
I don't think so, unless you can access the server via TCP and run Windows or macOS on it, which I don't think you can. Soundgrid is using some custom linux, if I recall correctly.

**Comment by melodiacman** [comment]
thanks. once i get it up and running i’ll see what i may be able to access. cheers!
